libraries/lib-project-file-io/DBConnection.cpp

```diff
@@ -268,7 +268,9 @@ bool DBConnection::Close()
       {
          using namespace std::chrono;
          std::this_thread::sleep_for(50ms);
-         pd->Pulse();
+         if (pd != nullptr) {
+             pd->Pulse();
```

Comment:
There is a `wxASSERT(pd)` few lines above. 
Why not make a stub implementation of `BasicUI`? There are lots of similar places where it's used

Reply:
Good idea, implemented empty QtBasicUI stub, no AU3 dependecies in current changes, please have a look